lms/djangoapps/lti_provider/tests/test_outcomes.py

```diff
@@ -363,3 +299,128 @@ def test_response_with_failing_status_field(self):
             major_code='<imsx_codeMajor>failure</imsx_codeMajor>'
         )
         self.assertFalse(outcomes.check_replace_result_response(response))
+
+
+class TestAssignmentsForProblem(ModuleStoreTestCase):
+    """
+    Test cases for the assignments_for_problem method in outcomes.py
+    """
+    def setUp(self):
+        super(TestAssignmentsForProblem, self).setUp()
+        self.course = CourseFactory.create()
+        self.course_key = CourseLocator(
+            org='some_org',
+            course='some_course',
+            run='some_run'
+        )
```

Comment:
Ah - I see what you mean. I can just use course.id and not bother with creating a course_key at all.